### DISCUSSION TITLE
Local replays should be available inside global scores

### DISCUSSION THREAD

**Discussion by Minorsonek** [ppy/osu#21705] [Ideas]
Hi, so I noticed it is very common for my replays not to be available when I click my score on global leaderboard, but when I go through local scores, then it is available and I can watch it. I think if the replay is accessible and stored locally, button to watch should be green regardless if I went through global or local scores

**Comment by peppy** [comment]
That's a bit of a hard one to make work without extra metadata, but going forward it's definitely a goal we have in mind.
